src/Lab_4/Lab_4.py

```diff
@@ -0,0 +1,47 @@
+class Perfume:
+    def __init__(self, volume=0, price=0.0, manufacturer="", public_numeric=0, public_string=""):
+        self.__volume = volume
+        self.__price = price
+        self.__manufacturer = manufacturer
+        self.public_numeric = public_numeric
+        self.public_string = public_string
+
+    def get_volume(self): 
+        return self.__volume
+
+    def set_volume(self, volume): 
+        self.__volume = volume
+
+    def get_price(self): 
+        return self.__price
+
+    def set_price(self, price): 
+        self.__price = price
+
+    def get_manufacturer(self): 
+        return self.__manufacturer
+
+    def set_manufacturer(self, manufacturer): 
+        self.__manufacturer = manufacturer
+
+    def __str__(self):
+        return f"{self.__manufacturer}: {self.__volume}ml, {self.__price} UAH"
```

Comment:
Please, Ensure there are two blank lines before the class definition and one blank line between methods. Additionally, keep lines under 79 characters for PEP 8 compliance.

Reply:
Fixed